signup-back/.env

```diff
@@ -1,5 +1,9 @@
-BACK_HOST=https://back.datapass-development.api.gouv.fr
-FRONT_HOST=https://datapass-development.api.gouv.fr
-OAUTH_HOST=https://auth-development.api.gouv.fr
-PG_USER=signup-back
-PG_PASSWORD=signup-back
+BACK_HOST=http://localhost:3000
+FRONT_HOST=http://localhost:3000
+OAUTH_HOST=https://auth-test.api.gouv.fr
+PG_USER=datapass-back
+PG_PASSWORD=datapass-back
+PG_HOST=db
+PG_DATABASE=datapass-development
```

Comment:
not found of this database being named as "signup-back" in every environment but here.

Reply:
same thing for PG_USER set to "back" in every other environments

Reply:
As you wish, I found that using the old name `signup` was a bad idea since we have the opportunity to change it here.
The database name is set in the docker-compose.yml file: https://github.com/betagouv/datapass/blob/dockerize/docker-compose.yml#L10

Reply:
It will be a pain for me for a while. it's just boring.